requirements.txt

```diff
@@ -36,3 +36,4 @@ numpy<=1.23
 rapidfuzz
 json-tricks==3.16.1
 onnx-simplifier>=0.3.6,<1.0
+data-gradients>=0.1.5
```

Comment:
I feel we should have a documentation page in SG describing this feature

Reply:
Maybe in a form of Notebook that will be exported to markdown 😉

Reply:
Yeah, that was the plan, notebook to markdown